src/Utils.js

```diff
@@ -0,0 +1,32 @@
+import Items from 'warframe-items';
+
+const find = (name, locale = 'en') => {
+  const items = new Items({
+    category: ['Skins', 'Primary', 'Secondary', 'Melee', 'Arch-Melee', 'Arch-Gun', 'Warframes', 'Archwing'],
+    i18n: locale,
+    i18nOnObject: true,
+  });
```

Comment:
i'd recommend moving the category array up a level out of find's scope so it doesn't recreate the array all the time.